### DISCUSSION TITLE
Connect pnp online error

### DISCUSSION THREAD

**Discussion by NaderHadjebi** [MicrosoftDocs/microsoft-365-community#486] [Q&A] [answered]
I am excited to delve in to some of these projects, however when I try to connect to pnp online
I run the following powershell "Connect-PnPOnline -Url https://mytenantname.sharepoint.com/"
then put in my username and password

 I get the following error

"Connect-PnPOnline : AADSTS50076: Due to a configuration change made by your administrator, or because you moved to a new location, you must use multi-factor authentication to access '00000003-0000-0ff1-ce00-000000000000'.
Trace ID: 7268c85a-ffb8-4213-b56d-c54a23c87501
Correlation ID: bd9f7e00-a991-442e-ae27-dcffbc94df9c

I am the global administrator and have checked and mfa is turned off

I have started researching it a bit, but was wondering if anyone has any insights that might help expedite an answer to this.
thanks!

**Comment by NaderHadjebi** [comment, marked as answer]
I trouble shot this and resolved the issue. I created another account for PnPPowershell with different authentication and it works now.
![pnlistSuccess](https://user-images.githubusercontent.com/60085977/126081249-19574b23-1f22-4d49-bc88-aea61614b0e5.GIF)

**Comment by sympmarc** [comment]
@NaderHadjebi I'm glad you solved your problem. For future questions about PnP.PowerShell, I'd suggest you post in the [in their repo](https://pnp.github.io/powershell/index.html#ive-found-a-bug-where-do-i-need-to-log-an-issue-or-create-a-pr).
